### ISSUE TITLE
Option to know number of running request in ollama

### ISSUE THREAD

**Issue by Jegatheesh001** [ollama/ollama#6904]
Option to know number of running request in ollama

**Comment by byarbrough** [comment]
@Jegatheesh001 please provide more detail on why you want this information. What problem are you trying to solve and how do you envision accessing "the number of requests running?"

**Comment by Gomez12** [comment]
For me I have wondered about this as well.
I have ollama setup so that it should run 8 in parallel, I have a client-program setup so that it does 8 parallel requests.

But the client also does things to the responses which also takes time, although the llm takes most of the time I am still left wondering if Ollama is really answering 8 requests, or only 6 because there are always 2 client-threads doing other things...

And with 1 client I could collect the info from the client but this becomes more complicated if you have multiple clients etc.

Basically I would like to know how efficient I am using the llm.

**Comment by jessegross** [comment]
I believe that this is the same idea as in #2004

Assuming so, let's track it there and add any additional information.

**Comment by Quadav** [comment]
> I believe that this is the same idea as in [#2004](https://github.com/ollama/ollama/issues/2004)
> 
> Assuming so, let's track it there and add any additional information.

I believe that these are 2 different things - amount of requests in concurrency that are running now, and the amount of requests that are waiting in the queue
